aad-app-manifest.json

```diff
@@ -48,6 +48,20 @@
     "legalAgeGroupRule": "Allow"
   },
   "passwordCredentials": [],
+  "preAuthorizedApplications": [
```

Comment:
I remember that it was that specific part that caused me having an error when trying to create the AAD app?

Reply:
Correct. There was an issue in CLI for M365, that's been fixed in the latest version